### PR TITLE
#565: added my name to credits again

### LINKED ISSUE #565: Andrew Tsai - Credits
### Description

Giving myself credit for the amazing developing work I will be doing at NER

### Acceptance Criteria

See my name in my favorite color on the credits page

### Proposed Solution

Add my name and favorite color to the "names" array in the CreditsPage.tsx file

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by andrewstsai**
## Changes

Added my name to the credits

## Notes

Had some trouble running the site, went through the software onboarding a second time from the start and made sure docker was connected but still getting an error stating that there's a failure to compile

## Test Cases

- Case A
- Edge case
- ...

## Screenshots

_If you made UI changes you must post a screenshot of the whole page for each change in 1) a normal sized window and 2) the smallest possible window_

_If you did any manual testing (e.g., with Postman), put screenshots of the http request and before and after of the db_

_If none of this applies, you can delete this section_

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `yarn.lock` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes #565

### CODE REVIEW

File: src/frontend/src/pages/CreditsPage/CreditsPage.tsx [hunk 1]
```diff
@@ -72,7 +72,8 @@ const CreditsPage: React.FC = () => {
     { name: 'Kenneth Wan', color: '#00FFFF' },
     { name: 'Madeline Engle', color: '#FF2D76' },
     { name: 'Parth Kabra', color: '#e53774' },
-    { name: 'Riley Platz', color: '#2d2fa6' }
+    { name: 'Riley Platz', color: '#2d2fa6' },
+    { name: 'Andrew Tsai', color: '3281a8'}
   ];
```
Comment (Peyton-McKee) [inline src/frontend/src/pages/CreditsPage/CreditsPage.tsx:77]: Make sure to add the # before the numbers for your color!

**Review by Peyton-McKee — CHANGES_REQUESTED** [review]
Great Work! One small change! Also, it looks like you forgot to fill out your pull request, make sure to explain the changes you made under the changes section, and add a screenshot of the localhost:3000/credits page with your name on it! Also delete any parts of the pull request that you dont end up using!

**Review by anthonybernardi — APPROVED** [review]
nice!!
